docs/specs/component.md

```diff
@@ -83,25 +83,16 @@ representing multiple endpoints.
 
 ## Instrumentation
 
-Vector components MUST be instrumented for optimal observability and monitoring.
-This is required to drive various interfaces that Vector users depend on to
-manage Vector installations in mission critical production environments. This
-section extends the [Instrumentation Specification].
-
-### Batching
+**Extends the [Instrumentation Specification].**
 
-For performance reasons, components SHOULD instrument batches of Vector events
-as opposed to individual Vector events. [Pull request #8383] demonstrated
-meaningful performance improvements as a result of this strategy.
+Vector components MUST be instrumented for optimal observability and monitoring.
 
 ### Events
 
-Vector implements an event driven pattern ([RFC 2064]) for internal
-instrumentation. This section lists all required and optional events that a
-component must emit. It is expected that components will emit custom events
-beyond those listed here that reflect component specific behavior.
-
-There is leeway in the implementation of these events:
+This section lists all required events that a component MUST emit. Additional events are
+It is expected that components will emit custom events beyond those listed here
+that reflect component specific behavior. There is leeway in the implementation
+of these events:
```

Comment:
```suggestion
This section lists all required events that a component MUST emit. Additional events
are listed that a component is RECOMMENDED to emit, but remain OPTIONAL. It is
expected that components will emit custom events beyond those listed here that
reflect component specific behavior. There is leeway in the implementation of these
events:

Reply:
Sorry, this was another casualty of one of my previous suggestions getting cut off.